src/CMakeLists.txt

```diff
@@ -136,45 +188,57 @@ blt_add_executable(
   lcals/TRIDIAG_ELIM.cpp
   lcals/TRIDIAG_ELIM-Seq.cpp
   lcals/TRIDIAG_ELIM-OMPTarget.cpp
-  polybench/POLYBENCH_2MM.cpp
-  polybench/POLYBENCH_2MM-Seq.cpp
-  polybench/POLYBENCH_2MM-OMPTarget.cpp
-  polybench/POLYBENCH_3MM.cpp
-  polybench/POLYBENCH_3MM-Seq.cpp
-  polybench/POLYBENCH_3MM-OMPTarget.cpp
-  polybench/POLYBENCH_ADI.cpp
-  polybench/POLYBENCH_ADI-Seq.cpp
-  polybench/POLYBENCH_ADI-OMPTarget.cpp
-  polybench/POLYBENCH_ATAX.cpp
-  polybench/POLYBENCH_ATAX-Seq.cpp
-  polybench/POLYBENCH_ATAX-OMPTarget.cpp
-  polybench/POLYBENCH_FDTD_2D.cpp
-  polybench/POLYBENCH_FDTD_2D-Seq.cpp
-  polybench/POLYBENCH_FDTD_2D-OMPTarget.cpp
-  polybench/POLYBENCH_FLOYD_WARSHALL.cpp
-  polybench/POLYBENCH_FLOYD_WARSHALL-Seq.cpp
-  polybench/POLYBENCH_FLOYD_WARSHALL-OMPTarget.cpp
-  polybench/POLYBENCH_GEMM.cpp
-  polybench/POLYBENCH_GEMM-Seq.cpp
-  polybench/POLYBENCH_GEMM-OMPTarget.cpp
-  polybench/POLYBENCH_GEMVER.cpp
-  polybench/POLYBENCH_GEMVER-Seq.cpp
-  polybench/POLYBENCH_GEMVER-OMPTarget.cpp
-  polybench/POLYBENCH_GESUMMV.cpp
-  polybench/POLYBENCH_GESUMMV-Seq.cpp
-  polybench/POLYBENCH_GESUMMV-OMPTarget.cpp
-  polybench/POLYBENCH_HEAT_3D.cpp
-  polybench/POLYBENCH_HEAT_3D-Seq.cpp
-  polybench/POLYBENCH_HEAT_3D-OMPTarget.cpp
-  polybench/POLYBENCH_JACOBI_1D.cpp
-  polybench/POLYBENCH_JACOBI_1D-Seq.cpp
-  polybench/POLYBENCH_JACOBI_1D-OMPTarget.cpp
-  polybench/POLYBENCH_JACOBI_2D.cpp
-  polybench/POLYBENCH_JACOBI_2D-Seq.cpp
-  polybench/POLYBENCH_JACOBI_2D-OMPTarget.cpp
-  polybench/POLYBENCH_MVT.cpp
-  polybench/POLYBENCH_MVT-Seq.cpp
-  polybench/POLYBENCH_MVT-OMPTarget.cpp
+  #Kokkos Bloc
+  lcals-kokkos/DIFF_PREDICT-Kokkos.cpp
+  lcals-kokkos/EOS-Kokkos.cpp
+  lcals-kokkos/FIRST_DIFF-Kokkos.cpp
+  lcals-kokkos/FIRST_MIN-Kokkos.cpp
+  lcals-kokkos/FIRST_SUM-Kokkos.cpp
+  lcals-kokkos/GEN_LIN_RECUR-Kokkos.cpp
+  lcals-kokkos/HYDRO_1D-Kokkos.cpp
+  lcals-kokkos/HYDRO_2D-Kokkos.cpp
+  lcals-kokkos/INT_PREDICT-Kokkos.cpp
+  lcals-kokkos/PLANCKIAN-Kokkos.cpp
+  lcals-kokkos/TRIDIAG_ELIM-Kokkos.cpp
+  #polybench/POLYBENCH_2MM.cpp
```

Comment:
Why are these all commented?

Reply:
The Polybench kernels are commented because we do not yet have Kokkos translations for them.  I believe the situation calls for additional logic to deal with the relative immaturity of the Kokkos design.  Does this make sense?  It's a choice David P and I made in the course of prototyping.